alpaca_trade_api/rest.py

```diff
@@ -899,40 +908,53 @@ def get_crypto_snapshot(self, symbol: str, exchange: str) -> SnapshotV2:
         return self.response_wrapper(resp, SnapshotV2)
 
     def get_news_iter(self,
-                      symbol: Union[str, List[str]],
+                      symbol: Optional[Union[str, List[str]]] = None,
                       start: Optional[str] = None,
                       end: Optional[str] = None,
-                      limit: int = None,
                       sort: Sort = Sort.Desc,
                       include_content: bool = False,
                       exclude_contentless: bool = False,
+                      total_limit: int = None,
+                      no_total_limit: bool = False,
```

Comment:
A would just go with `limit: int = 10` instead for
- simplicity
- consistency with the other endpoints